workflows/variantbenchmarking.nf

```diff
@@ -64,14 +64,29 @@ workflow VARIANTBENCHMARKING {
 
         regions_bed_ch = params.regions_bed ? Channel.fromPath(params.regions_bed, checkIfExists: true).collect()
                                                     : Channel.empty()
+        targets_bed_ch = params.targets_bed ? Channel.fromPath(params.targets_bed, checkIfExists: true).collect()
+                                                    : Channel.empty()
     }else{
         log.error "Please specify params.truth_id and params.truth_vcf or params.regions_bed to perform benchmarking analysis"
         exit 1
     }
 
+    // Optional files for Happy or Sompy
+    falsepositive_bed   = params.falsepositive_bed  ? Channel.fromPath(params.falsepositive_bed, checkIfExists: true).map{ bed -> tuple([id: "falsepositive"], bed) }.collect()
```

Comment:
Just so I understand, are you using `.collect()` just to convert to a value channel? `.first()` is used often for that as well (but probably doesnt matter).

Reply:
yes, I am using .collect() just to convert the channel. I will keep this in my mind. thank you :)